src/types/index.ts

```diff
@@ -65,7 +65,7 @@ export type CheckRun = EmitterWebhookEvent<'check_run'>['payload']['check_run'];
 //
 // Need to do this as we can't convert a string literal union to an array of literals
 export const CHECK_RUN_PROPERTIES = ['id', 'head_sha', 'html_url'] as const;
-export type CheckRunProperty = typeof CHECK_RUN_PROPERTIES[number];
+export type CheckRunProperty = (typeof CHECK_RUN_PROPERTIES)[number];
```

Comment:
Why this change ?

Reply:
seems like an autoformatting change I didn't make this change